### PR TITLE
Conditionally render necessary components on home for logged in donor portal user

### LINKED ISSUE #94: Donate button on the donate-now page does not work

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose

The purpose of this PR is to fix #94

## Goals

To implement the logged in user views on donor portal home

## Approach

- Created a context to keep the userid on the topmost component of the donor app
- Passed that down by getting the value from local storage into the home page 
- Conditionally rendered my pledges and logout button on nav bar using dummy data

- Removed the image from aid package card ([Refer meeting notes](https://docs.google.com/document/d/1VkueFsBJTPBadynX6kY4_RTgfT2NCVVTcCdurOF_bfU/edit#heading=h.wbyxqrju4amh)) 

### Screenshots

Logged in view of the home page
image

## Related PRs
 
#45 
Note: Didn't update the `DonorAidPackage` data structure because it's already updated on this PR

### CODE REVIEW

File: donor-portal/src/pages/home/index.tsx [hunk 1]
```diff
@@ -74,23 +77,46 @@ const packageList = [
...
-          return donorPackage.pledgedPercentage === 100;
-        }
-      })
+    const aidPackages = packageList;
+    setGoalReachedAidPackages(
+      aidPackages.filter((aidPackage) => aidPackage.pledgedPercentage == 100)
+    );
+    setGoalPendingAidPackages(
+      aidPackages.filter((aidPackage) => aidPackage.pledgedPercentage != 100)
     );
```
Comment (jayasanka-sack) [inline donor-portal/src/pages/home/index.tsx:114]: Move this part to a seperate effect as it doesn't depend on the `userId`

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: donor-portal/src/pages/home/index.tsx (hunk 1 above)
Comment (Gravewalker666) [reply donor-portal/src/pages/home/index.tsx:114]: Sure

Comment (Gravewalker666) [reply donor-portal/src/pages/home/index.tsx:121]: Fixed with the commit 751be2d, Please review

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Gravewalker666 !

**Pull request merged**